src/test/java/io/turntabl/ClientRegisterTest.java

```diff
@@ -13,7 +13,7 @@
     Client client3 = new Client(003, "Yaa Nuamah", "Dansoman", "+23355345572", "yaa@gmail.com", ClientLevel.PREMIUM);
     Client client4 = new Client(004, "Sam Bartels", "Ashaiman", "+23355554572", "sam@gmail.com", ClientLevel.GOLD);
     Client client5 = new Client(005, "Sam Moorehouse", "Dansoma", "+233553458272", "Moore@gmail.com", ClientLevel.PLATINUM);
-    Client client6 = new Client(006, "James Barsely", "London", "+233553454472", "james96@gmail.com", ClientLevel.PREMIUM);
+    Client client6 = new Client(006, "Isaac Barsely", "London", "+233553454472", "james96@gmail.com", ClientLevel.PREMIUM);
```

Comment:
Why did you change James to Isaac?

Reply:
I used it for a unit test, thanks for asking mate :)

Reply:
Okay!
That's fine. LGTM

Reply:
Alright that's awesome.